tool/mcptoolset/set.go

```diff
@@ -48,15 +48,20 @@ import (
 //		},
 //	})
 func New(cfg Config) (tool.Toolset, error) {
+	if cfg.Client == nil {
+		cfg.Client = mcp.NewClient(&mcp.Implementation{Name: "adk-mcp-client", Version: "v1.0.0"}, nil)
+	}
 	return &set{
-		client:     mcp.NewClient(&mcp.Implementation{Name: "adk-mcp-client", Version: "v1.0.0"}, nil),
```

Comment:
Could I ask to replace `"v1.0.0"` with adk-go baked-in version for a proper client tagging https://github.com/google/adk-go/blob/99924140926575895867a1d2f22afbcc7492f973/internal/version/version.go#L18